src/tls13utils.rs

```diff
@@ -59,9 +59,7 @@ pub const PARSE_FAILED: TLSError = 133;
 pub const INSUFFICIENT_DATA: TLSError = 134;
 
 pub fn error_string(c: u8) -> String {
```

Comment:
This will have to go anyway.

Reply:
Okay, but not in this PR, right? Here, only Clippy suggestions are applied.